src/services/filedrop.js

```diff
@@ -103,10 +105,11 @@ export async function uploadFileDrop(folderId, fileDrop, lockToken, shareToken)
 		{
 			headers: {
 				'x-e2ee-supported': true,
-				'e2e-token': lockToken,
+				...(encryptionVersion === 2 ? { 'e2e-token': lockToken } : {}),
 			},
 			params: {
 				shareToken,
+				...(encryptionVersion === 1 ? { 'e2e-token': lockToken } : {}),
```

Comment:
Should this be version 2 instead? Is the `e2e-token` supported in v1?

```suggestion
...(encryptionVersion === 2 ? { 'e2e-token': lockToken } : {}),
```

Reply:
Yes, e2e token is the lock token. It was passed as a param in v1, and is now pased as a header in v2.